wordpress-mobile/WordPress-iOS
# [Dark Mode] Fix the Fancy Alert View background colors

Refs. #12320 and part of [#40](https://github.com/wordpress-mobile/WordPressUI-iOS/pull/40)

This PR fixes the white background displayed in the Fancy Alerts on iOS 13 with Dark Mode enabled.

![fancy-alert-dark-mode](https://user-images.githubusercontent.com/912252/64178279-161e3380-ce59-11e9-96e3-ffa478149a29.jpg)


## To test:
• Run the branch on a device/simulator with iOS 13 - Dark Mode enabled
• Check the Fancy Alert doesn't have a white background
• Run the branch on iOS 12 to check everything works normally 

_I will update the Podfile as soon [#40](https://github.com/wordpress-mobile/WordPressUI-iOS/pull/40) will be merged._

This PR doesn't fix #12441

Update release notes:
